omod_1.x/src/main/webapp/bill.jsp

```diff
@@ -17,8 +16,8 @@
 <openmrs:require allPrivileges="<%=PrivilegeWebConstants.BILL_PAGE_PRIVILEDGES%>" otherwise="/login.htm" redirect="/module/openhmis/cashier/bill.form" />
 
 <%-- Force our newer jquery version to load first --%>
-<openmrs:htmlInclude file="/moduleResources/openhmis/backboneforms/js/lib/jquery.js" />
 <%@ include file="/WEB-INF/template/header.jsp"%>
+<openmrs:htmlInclude file="/moduleResources/openhmis/backboneforms/js/lib/jquery.js" />
```

Comment:
i tested it and it works fine and does not break any jquery functionality. The older jquery is loaded when the localheader is included isn't??